### DISCUSSION TITLE
Will RVC work with GTX?

### DISCUSSION THREAD

**Discussion by AsahiTokunaga** [w-okada/voice-changer#1168] [Q&A] [answered]
I want to use rvc with Nvidia Geforce GTX series gpu.

**Comment by type4101** [comment, marked as answer]
Unlikely.  It doesn't really work on most systems.  The Passthrough will work (you can hear your own voice or the original recording) but when you try to use the voices, it doesn't work.  This is true whether you use a microphone or an audio file as the source.  W-Okada Voice Changer just doesn't pick up the sound.  Hopefully someday the bugs can get fixed, but if you look at the discussion posts and bug-fix requests, you will see people begging for help, but no response from the developer.  For months and months.  If you get it working, please post and let us know how you fixed the no-voice issue!
